src/lib/components/lines-and-dots/svg/group-details-row.svelte

```diff
@@ -77,96 +47,117 @@
 
   $: childWorkflowStartedEvent =
     group && group.eventList.find(isChildWorkflowExecutionStartedEvent);
+
+  $: duration = formatDistanceAbbreviated({
+    start: group?.initialEvent?.eventTime,
+    end: group?.lastEvent?.eventTime,
+    includeMilliseconds: true,
+  });
+
+  $: {
+    if (group?.pendingActivity) {
+      if (group.pendingActivity.attempt > 1) {
+        status = translate('events.event-classification.retrying');
+      } else {
+        status = translate('events.event-classification.pending');
+      }
+    }
+  }
+
+  const onDecode = async () => {
+    await tick();
+    contentHeight = innerContent?.offsetHeight;
+  };
 </script>
 
-<g role="button" tabindex="0" class="relative z-50 cursor-pointer">
-  <Box point={[x, y]} {width} height={boxHeight} fill="#465A78" />
-  <Box point={[x, y]} {width} {height} fill="#1E293B" />
-  <foreignObject {x} {y} {width} height={fontSizeRatio}>
-    <div class="flex h-full items-center justify-between text-sm text-white">
-      <div class="flex h-full items-center gap-2">
-        <div
-          class="px-4 py-1 text-black"
-          style="background-color: {getStatusColor(status)};"
-        >
-          {status ? spaceBetweenCapitalLetters(status) : group.label}
+<g role="button" tabindex="0" class="relative z-50">
+  <foreignObject {x} {y} {width} height={contentHeight}>
+    <div bind:this={innerContent} class="flex flex-col border-b border-subtle">
```

Comment:
Couldn't you use `bind:offsetHeight` instead here? https://svelte.dev/docs/svelte/bind#Dimensions

Reply:
👍